### PR TITLE
Docs: Updating documentation for `angular_support_enabled`

### PR DESCRIPTION
**Pull request by tolzhabayev**
**What is this feature?**
As pointed out [here](https://github.com/grafana/grafana/issues/88739#issuecomment-2149321558) the documentation is incorrect, this PR fixes it.

### CODE REVIEW

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 1]
```diff
@@ -705,7 +705,7 @@ Set the policy template that will be used when adding the `Content-Security-Poli
 
 ### angular_support_enabled
 
-This currently defaults to `true` but will default to `false` in a future release. When set to false the angular framework and support components will not be loaded. This means that
+When set to false the angular framework and support components will not be loaded. This means that
```
Comment (sympatheticmoose) [inline docs/sources/setup-grafana/configure-grafana/_index.md:708]:
```suggestion
This is set to false by default, meaning that the angular framework and support components will not be loaded. This means that
```
Removes ambiguity that default is true

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 2]
```diff
@@ -705,7 +705,7 @@ Set the policy template that will be used when adding the `Content-Security-Poli
 
 ### angular_support_enabled
 
-When set to false the angular framework and support components will not be loaded. This means that
+This is set to false by default, meaning that the angular framework and support components will not be loaded. This means that
```
Comment (josmperez) [inline docs/sources/setup-grafana/configure-grafana/_index.md:708]:
```suggestion
This flag is set to false by default, meaning that the AngularJS framework and support components will not be loaded. This means that
```

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 3]
```diff
@@ -705,7 +705,7 @@ Set the policy template that will be used when adding the `Content-Security-Poli
...
 all [plugins]({{< relref "../../developers/angular_deprecation/angular-plugins" >}}) and core features that depend on angular support will stop working.
```
Comment (josmperez) [inline docs/sources/setup-grafana/configure-grafana/_index.md:709]:
```suggestion
all [plugins]({{< relref "../../developers/angular_deprecation/angular-plugins" >}}) and core features that depend on AngularJS support will stop working.
```

**Review by josmperez — APPROVED** [review]
A couple of nits, but LGTM.

**Pull request merged**